<ui version="4.0" >
 <class>CashierReports</class>
 <widget class="QDialog" name="CashierReports" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cashier Reports</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../data/icons/pyKafe.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" colspan="2" >
    <widget class="QDoubleSpinBox" name="doubleSpinBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Income:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QDateTimeEdit" name="dateTimeEdit_2" />
     </item>
     <item rowspan="2" row="0" column="1" >
      <widget class="QToolButton" name="toolButton" >
       <property name="text" >
        <string>Find</string>
       </property>
       <property name="icon" >
        <iconset>../../data/icons/find.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QDateTimeEdit" name="dateTimeEdit" />
     </item>
    </layout>
   </item>
   <item row="2" column="2" >
    <widget class="QSpinBox" name="spinBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QSpinBox" name="spinBox_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Members deleted:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Members added:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CashierReports</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CashierReports</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>CashierReports</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>218</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>132</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
